docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -2,6 +2,16 @@
 
 As part of the Auth SDK module, there is a self defence check feature.  This feature offers multiple security checks that can be used to analyse the secuirty of the device.  This can be useful for apps to detect vulnerablities and react accordingly.  For example, if the app detects security issue *X* then do *Y*, where *X* could be no lock screen is set and *Y* is show a system prompt to setup the lock screen.
 
+Self-defence checks to detect a possible compromised runtime environment that can expose a mobile application to more vectors of attack that can lead to data leaks.
```

Comment:
Maybe refactor the second sentence above to use this  - " As part of the Auth SDK module, there is a self defence check feature. These self defence checks can be used to detect a possible compromised runtime environment that can expose an app to more vectors of attack that can lead to data leaks."

I still find the sentence kind of long though

Reply:
I think there might be some duplication in this section with the existing intro above.  The only differing info I can see is this sentence:

`Integration with the AeroGear Services Metrics SDK to report self-defence checks data.`